platform/platform-configuration-migration/src/test/java/org/codice/ddf/configuration/admin/ConfigurationAdminMigrationTest.java

```diff
@@ -82,25 +80,16 @@
 
     private Path configFilesExportPath = Paths.get("/root/etc/exported/etc");
 
-    @Mock
-    private Path processedDirectoryPath;
-
-    @Mock
-    private File processedDirectory;
+    private Path processedDirectoryPath = Paths.get("/root/etc/processed");
 
-    private Path configFileInProcessedDirectory =
-            Paths.get("/path/to/processed/myConfigFile.config");
+    private Path configFileInProcessedDirectory = Paths.get("/root/etc/processed/pid.config");
```

Comment:
Maybe we should do `processedDirectoryPath.resolve(CONFIG_FILENAME)` instead? That way if we change the path and/or file name, we only need to change it in one place.

Same comment applies for `configFileInFailedDirectory`.